src/rpc/types.zig

```diff
@@ -151,3 +151,19 @@ pub const RpcVersionInfo = struct {
 };
 
 pub const Signature = []const u8;
+
```

Comment:
More of a question: What goes into `src/rpc/types.zig` and what goes into `src/rpc/client.zig`. On a cursory look it looks as if the RPC responses are defined in `src/rpc/types.zig` while inpurs are defined inline in `src/rpc/client.zig`

Would that be a correct read of the pattern? If so, what the benefit to that vs having all types (both input and output) defined in `src/rpc/types.zig`

Reply:
I don't know why it's organized this way. I agree with you about reorganizing this. I've been messing with this in my spare time and I have `methods` file that defines all the request and response types, and associates the types with each other. I'll probably open a PR with this pretty soon.